### DISCUSSION TITLE
Number of harmonic IFCs

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#155] [Q&A] [answered]
Dear Dr. Tadano,

With the suggest mode of alm, I'm getting the same number of displacement patterns in the harmonic approximation, regardless of the cutoff. When I decrease the cutoff, the number of free harmonic IFCs decreases as well (as it should), but the number of displacement patterns remains the same. Is this a normal thing to happen?

Thanks in advance,
Ricardo.

**Comment by ttadano** [comment, marked as answer]
Yes, it is quite normal. For the harmonic force constants, the number of displacement patterns does not increase when the cutoff radius is increased.

**Reply by RicardoCecconello** [reply]
Right, thank you!
